packages/components/src/horizontal-bar-list/stats-card.scss

```diff
@@ -116,6 +116,11 @@
 			grid-row: 1;
 		}
 
+		.stats-card__content--hero {
+			grid-column: auto;
+			grid-row: auto;
+		}
```

Comment:
At first I tried to just drop the `grid-column` and `grid-row` values from the `.stats-card__content` class, which ended up breaking the other cards, hence why a new class was added.